.pylintrc

```
@@ -371,6 +536,15 @@ min-public-methods=2
 
 [IMPORTS]
 
+# List of modules that can be imported at any level, not just the top level
+# one.
+allow-any-import-level=clib.Session,
```

```suggestion
allow-any-import-level=clib,
```
Maybe I should change this?

---

It's all up to you, as I know very little about pylint.

---

Lemme downgrade my pylint and see if it works.

---

Then it breaks pylint 2.5.x... I've allowed for any import level for both `clib` and `clib.Session` so that pylint 2.4 and pylint 2.5 will work. Will merge this branch once the CI passes.